src/main/kotlin/com/sourcegraph/cody/edit/EditCodeInlayRenderer.kt

```diff
@@ -0,0 +1,61 @@
+package com.sourcegraph.cody.edit
+
+import com.intellij.openapi.editor.EditorCustomElementRenderer
+import com.intellij.openapi.editor.Inlay
+import com.intellij.openapi.editor.markup.TextAttributes
+import com.intellij.ui.JBColor
+import java.awt.Graphics
+import java.awt.Rectangle
+import javax.swing.CellRendererPane
+import javax.swing.JPanel
+
+class EditCodeInlayRenderer(private val component: JPanel, private val requiredHeight: Int) :
```

Comment:
Sorry to be dense, but where is this used?

Reply:
Was a leftover, not used - removed.